src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```
@@ -149,6 +149,28 @@ Services
     }
   }
   ##END
+  ##BEGIN TornadoSandboxStore
+  TornadoSandboxStore
+  {
+    Port = 9196
```

No Port, but protocol:

```suggestion
    Protocol = https
```

---

Yes, corrected